src/harness/testcases/testdata/ppa_record_3.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "PPA Zone 1",
+  "creator": "admin_0",
+  "terminated": false,
+  "usage": "PPA",
+  "zone": {
+    "type": "FeatureCollection",
+    "features": [
+      {
+        "type": "Feature",
+        "properties": {},
+        "geometry": {
+          "type": "Polygon",
+          "coordinates": [
```

Comment:
This is wrong I believe, missing bracket.
A polygon is defined as:
  "coordinates":   [       => the list of loop, firsdt being exterior, other being holes
      [  => one loop
          [40, 50]  => a coordinate
          ...
      ],   => end of loop one
     [  => another loop being a hole
         ...